src/main/java/com/alkemy/ong/config/security/JwtRequestFilter.java

```diff
@@ -1,4 +1,74 @@
 package com.alkemy.ong.config.security;
 
-public class JwtRequestFilter {
+import io.jsonwebtoken.*;
+import io.jsonwebtoken.io.Encoders;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class JwtRequestFilter extends OncePerRequestFilter {
+
+    @Autowired
```

Comment:
No usar inyeccion de dependencia con `@Autowired` es una mala práctica. Hacer el atributo private final y agregar a la clase` @RequiredArgsConstructor`.
VER: https://dzone.com/articles/spring-di-patterns-the-good-the-bad-and-the-ugly